src/datagrid/DataGridValues.tsx

```diff
@@ -94,6 +103,10 @@ export const customRows = [
     },
 ];
 
+/**
```

Comment:
Observation: When left style parameter is set to pixels for the filter manually then on resizing the browser the position of the filter  changes such that it can be anywhere on the browser.

Reply:
@payald Thanks for noticing this. It is fixed now